src/main/java/org/prebid/server/auction/TargetingKeywordsCreator.java

```diff
@@ -192,7 +192,7 @@ private static PriceGranularity convertToCustomPriceGranularity(String stringPri
         if (isApp) {
             keywordMap.put(HB_ENV_KEY, HB_ENV_APP_VALUE);
         }
-        if (winningBid && winUrl != null) {
+        if (!isVideo && winningBid && winUrl != null) {
```

Comment:
It's kind of architectural decision. For sure, `isVideo` is definitely a marker for skipping `HB_WINURL_KEY`, but adding new argument `boolean isVideo` has something like duplication sense since we already have `winUrl`.

So, my propose is: don't modify `TargetingKeywordsCreator` and pass `winUrl` as NULL in `BidResponseCreator.java.toBid(...)` method. Furthermore, we won't create `winUrl` for video bids at all.

What do you think?